ext_localconf.php

```diff
@@ -2,12 +2,19 @@
 defined('TYPO3_MODE') or die();
 
 $initialize = function ($extKey) {
+    $isVersion9Up = \TYPO3\CMS\Core\Utility\VersionNumberUtility::convertVersionNumberToInteger(TYPO3_version) >= 9000000;
```

Comment:
Is there a reason not to use [version_compare()](https://www.php.net/manual/de/function.version-compare.php) here?

Reply:
Hi, sorry didn't recognized this... I'm normally using it in this way...

Reply:
I would suggest to use `version_compare` here and use it directly in the if e.g.:
```
if (version_compare(TYPO3_branch, '9.5', '>=')) {
```
And we should mention the version where the feature got introduced to not break anything ☺️ 

If I have a look at the ChangLog for v9, the expression language was introduced with 9.4, IIUC: https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.4/Feature-85828-MoveSymfonyExpressionLanguageHandlingIntoEXTcore.html

Reply:
Hi, thanks for the fast reply, now I could understand it.
I changed it now to version_compare with TYPO3 branch.
See e35da1d

Reply:
Looks good so far. I'm asking myself whether we need this line still:

`$isVersion9Up = \TYPO3\CMS\Core\Utility\VersionNumberUtility::convertVersionNumberToInteger(TYPO3_version) >= 9000000;`